README.md

```diff
@@ -51,6 +51,11 @@ make juno
 ./build/juno
 ```
 
+### Build and Run with Docker
+
+```shell
+docker run --name juno -it -p 6060:6060 nethermindeth/juno
```

Comment:
Let's remove it from here other we will have to pass the port to juno flags. Unless we use a config file

Reply:
Why? 6060 is the default right?

Reply:
They need to map their db path as well, am I right @wojciechos ?

Reply:
Yeah, 6060 is default. Ok, so as it is configurable I'll remove those.

Reply:
Unless we are ok with default paths. The best solution to this is to have a config file. We use `yaml` format and examples of it can be found [here](https://github.com/NethermindEth/juno/blob/develop/cmd/juno/juno_test.go)

Reply:
I've adjusted docker usage with passing all juno configurations including rpc-port and db-path.